lib/active_merchant/billing/gateways/cyber_source.rb

```diff
@@ -941,6 +941,16 @@ def in_fraud_review?(response)
       def success?(response)
         response[:decision] == @@decision_codes[:accept]
       end
+
+      def message_from(response)
+        if response[:reasonCode] == '101' && response[:missingField]
+            response[:message] + ': ' + response[:missingField]
```

Comment:
Could you use string interpolation for these instead of concatenation?
```suggestion
            "#{response[:message]}: #{response[:missingField]}"
```

Reply:
https://github.com/activemerchant/active_merchant/blob/63233b7e6c85f785d1636a6da0a1b42eda1ea2bd/lib/active_merchant/billing/gateways/cyber_source.rb#L947